ts/features/lollipop/index.ts

```diff
@@ -0,0 +1,59 @@
+import { PublicKey } from "@pagopa/io-react-native-crypto";
+import URLParse from "url-parse";
+import { SignatureAlgorithm } from "../../utils/httpSignature/types/SignatureAlgorithms";
+import { SignatureComponents } from "../../utils/httpSignature/types/SignatureComponents";
+import { KeyInfo } from "./../../utils/crypto";
+
+export type LollipopConfig = {
+  keyInfo: KeyInfo;
+  nonce: string;
+  customSignatures?: ReadonlyArray<string>;
+};
+
```

Comment:
I imagine that `keyInfo` is related to the **public key**, which is information that exists within the domain of the app, is immutable, and is not known to the single product. The `nonce`, on the other hand, varies from call to call and is related to the **product domain**. As a result, the calling function (of the product) may be aware of the `nonce` to pass but is unaware of `keyInfo` (as well as `customSignatures`). I propose splitting these two things and obtaining the signature information (`keyInfo`) in a different way depending on how the login (and thus key generation) has been implemented.

Reply:
`KeyInfo` is now out of the LollipopConfig. We moved it to the BackendClient constructor.
Regarding `customSignatures`, is the same concept made for `nonce`. They may vary from call to call and it's up to the caller to define them (or not)